apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -3667,13 +3326,17 @@ static ItemComponent mapEobCommonItemCarrierDME(
 
       item.setServiced(
           new Period()
-              .setStart((convertToDate(firstExpenseDate.get())), TemporalPrecisionEnum.DAY)
-              .setEnd((convertToDate(lastExpenseDate.get())), TemporalPrecisionEnum.DAY));
+              .setStart(
+                  (CommonTransformerUtils.convertToDate(firstExpenseDate.get())),
+                  TemporalPrecisionEnum.DAY)
+              .setEnd(
+                  (CommonTransformerUtils.convertToDate(lastExpenseDate.get())),
+                  TemporalPrecisionEnum.DAY));
```

Comment:
I think a static import of these methods would make the change set smaller and show that not much changed.

Reply:
done...thought about it at first and didn't think there was that many...when I just implemented using static I was surprised at the count!